crates/iota-config/src/local_ip_utils.rs

```diff
@@ -18,13 +18,20 @@ pub struct SimAddressManager {
 }
 
 #[cfg(msim)]
-impl SimAddressManager {
-    pub fn new() -> Self {
+impl Default for SimAddressManager {
+    fn default() -> Self {
```

Comment:
I wonder to what extent Clippy actually complains about new without default. Sometimes new shouldn't be default I guess.

Reply:
Why shouldn't it be the default, assuming there are no params in `new`?

Reply:
I don't know, just thinking out loud, don't mind me.
A expensive new that would be different from default idk...